haproxy_exporter.go

```diff
@@ -352,6 +375,27 @@ loop:
 	return 1
 }
 
+func (e *Exporter) parseInfo(i *io.ReadCloser, ch chan<- prometheus.Metric) error {
```

Comment:
nit: the function doesn't close the reader, so requiring an `io.Reader` interface would be more appropriate. Also no pointer receiver necessary for interfaces.

Reply:
Thanks, fixed.